diff --git a/node_modules/react-native-calendars/src/expandableCalendar/agendaList.d.ts b/node_modules/react-native-calendars/src/expandableCalendar/agendaList.d.ts
index a2c623e..37a843b 100644
--- a/node_modules/react-native-calendars/src/expandableCalendar/agendaList.d.ts
+++ b/node_modules/react-native-calendars/src/expandableCalendar/agendaList.d.ts
@@ -22,6 +22,8 @@ export interface AgendaListProps extends SectionListProps<any, DefaultSectionT>
     viewOffset?: number;
     /** enable scrolling the agenda list to the next date with content when pressing a day without content */
     scrollToNextEvent?: boolean;
+
+    renderSectionHeader?: any;
 }
 /**
  * @description: AgendaList component
