diff --git a/node_modules/react-native-calendars/src/expandableCalendar/Context/Provider.js b/node_modules/react-native-calendars/src/expandableCalendar/Context/Provider.js
index a6d4af1..3a17c4b 100644
--- a/node_modules/react-native-calendars/src/expandableCalendar/Context/Provider.js
+++ b/node_modules/react-native-calendars/src/expandableCalendar/Context/Provider.js
@@ -7,6 +7,8 @@ import { useDidUpdate } from '../../hooks';
 import { UpdateSources } from '../commons';
 import styleConstructor from '../style';
 import CalendarContext from './index';
+import MaterialCommunityIcon from "react-native-vector-icons/MaterialCommunityIcons";
+
 import { shouldAnimateTodayButton, shouldAnimateOpacity, getButtonIcon, getPositionAnimation, getOpacityAnimation, getTodayDate, getTodayFormatted } from './Presenter';
 const TOP_POSITION = 65;
 /**
@@ -91,8 +93,13 @@ const CalendarProvider = (props) => {
     const renderTodayButton = () => {
         return (<Animated.View style={[style.current.todayButtonContainer, { transform: [{ translateY: buttonY.current }] }]}>
         <TouchableOpacity style={[style.current.todayButton, todayButtonStyle]} onPress={onTodayPress} disabled={isDisabled}>
-          <Animated.Image style={[style.current.todayButtonImage, { opacity: opacity.current }]} source={buttonIcon}/>
-          <Animated.Text allowFontScaling={false} style={[style.current.todayButtonText, { opacity: opacity.current }]}>
+        <MaterialCommunityIcon
+            name={"calendar-today"}
+            size={21}
+            color="white"
+            style={{ marginRight: 5 }}
+          />
+          <Animated.Text allowFontScaling={false} style={[style.current.todayButtonText, { opacity: opacity.current, textTransform:"uppercase" }]}>
             {today.current}
           </Animated.Text>
         </TouchableOpacity>
